src/services/side-panels-manager/models/item-wrapper.tsx

```diff
@@ -70,31 +81,6 @@ export class ItemWrapper {
     });
   }
 
-  private injectIconComponent(onClick: (panelItemId: number) => void): void {
```

Comment:
what was this replaced with ?

Reply:
https://github.com/kaltura/playkit-js-ui-managers/blob/0ed6eca69f8112b0c79d9859623571db06692a21/src/services/side-panels-manager/models/item-wrapper.tsx#L30
the handling moved to the upper bar manager service